compose/aarch64.yml

```diff
@@ -2,7 +2,7 @@
 
 services:
   cardano-db-sync:
-    image: ghcr.io/input-output-hk/ogmios-tracker/cardano-db-sync:${CARDANO_DB_SYNC_VERSION:-sancho-4-0-0}
+    image: ghcr.io/input-output-hk/ogmios-tracker/cardano-db-sync:${CARDANO_DB_SYNC_VERSION:-sancho-4-2-1}
```

Comment:
Published: [this image](https://github.com/input-output-hk/ogmios-tracker/pkgs/container/ogmios-tracker%2Fcardano-db-sync/205957380?tag=sancho-4-2-1).